cpp/ps0003/ps0003_sol.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+int main()
+{
+    // prompting user for number 2/2
+    int number;
+    std::cout << "please enter the number\n";
+    std::cin >> number;
+    
+    // checking if the number is odd or even 3/3
+    if (number % 2 == 0) 
+        std::cout << "the number is even\n";   
+    else 
+        std::cout << "the number is odd\n";
+    
+    return 0;
+}
+// great work 😎 8/10 ⭐
```

Comment:
- [x] program returns `0` as everything is okay
- [ ] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code